src/styles.css

```diff
@@ -1 +1,28 @@
 /* You can add global styles to this file, and also import other style files */
+html {
+    font-size: 62.5%; /* Now 10px = 1rem! */
+}
+
+body{
+    background: linear-gradient(#4be1f2, #0012d8);
```

Comment:
@rgavinc @jcwheatley With a blue background like this you'll probably want to change the link color as well?